bfss3/bfss3.go

```diff
@@ -444,3 +450,30 @@ func (i *iterator) fetchNextPage() error {
 	}
 	return nil
 }
+
+// --------------------------------------------------------------------
+
+// NewHTTPClient returns a copy of net/http.DefaultClient with implicit GZIP compression disabled.
+//
+// It is intended to be used as aws.Config.HTTPClient.
+func NewHTTPClient() *http.Client {
```

Comment:
I don't really like the idea of making this public, but I think it's a must for `aws.Config`.
But I couldn't default it in normalisation, as `aws.Config` is a non-pointer field of `bfss3.Config`.

Reply:
Actually, sec... I'll change this, as `aws.Config.HTTPClient` is nil-able field, which we can default to this.